flow-libs/htmlnano.js.flow

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+declare module 'htmlnano' {
+  declare type Options = {|
+    minifyJs?: boolean,
+    minifyCss?: boolean,
+    userConfig?: mixed,
```

Comment:
```suggestion
```

`userConfig` is definitely not an option

Reply:
Ah my mistake, we were passing in a config of type any. I made it inexact now and removed it...